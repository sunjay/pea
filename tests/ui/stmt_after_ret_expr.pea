fn foo(b) {
    let a = b % 3;
    a
    println!(a);
}
