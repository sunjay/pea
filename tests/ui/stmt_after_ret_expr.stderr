[tests/ui/stmt_after_ret_expr.pea:4:5-4:11] error: expected `}`, found: `println`

error: aborting due to 1 previous error

